ironfish-cli/src/commands/deposit-all.ts

```diff
@@ -227,13 +227,13 @@ export default class DepositAll extends IronfishCommand {
       const sendableOre = confirmedBalance - fee
 
       // terminate condition
-      if (terminate && pendingBalance < sendableOre + fee) {
+      if (terminate || pendingBalance < fee) {
```

Comment:
The condition here isn't quite right.

It needs to be `&&`: `terminate` is a command-line flag, so if it is set to true then this change would result in termination after the first deposit.

The check here doesn't account for `minDepositOre`: if the pending balance is less than `minDepositOre + fee` then no more deposits can be made.